src/Psalm/Type/Atomic/TArray.php

```diff
@@ -14,6 +14,7 @@ class TArray extends \Psalm\Type\Atomic
 
     /**
      * @var array{\Psalm\Type\Union, \Psalm\Type\Union}
+     * @psalm-suppress NonInvariantPropertyType
```

Comment:
Maybe this and similar properties can be deleted from the code and we can just use the properties declared in the parent classes?